aws/policy/application-services.yaml

```diff
@@ -163,6 +163,14 @@ Statement:
       - logs:PutMetricFilter
     Resource:
       - 'arn:aws:logs:{{ aws_region }}:{{ aws_account_id }}:log-group:ansible-test*'
+  - Sid: DescribeEventRules
```

Comment:
Could you please move these Actions and Resource to `AllowGlobalResourceRestrictedActionsWhichIncurNoFees`?  We often run into policy size limits and have to do creative refactoring, this change doesn't need its own Sid.

Reply:
Sure, done.